src/email-template/email-template.service.ts

```diff
@@ -118,6 +122,10 @@ export class EmailTemplateService {
           }
         }
 
+        if (!renderParams.template) {
```

Comment:
anna because of this check it happens that no daily messages is being sent at all. hence in class daily message template of renderParams is never being set... can you check the logic again? @EeroHeikkinen @KamupakEero